### PR TITLE
Migrate pylint to 1.7

### PR DESCRIPTION
**Pull request by mohierf**

### CODE REVIEW

File: alignak/dependencynode.py [hunk 1]
```diff
@@ -151,20 +151,20 @@ def get_state(self, hosts, services):
             return self.get_host_node_state(host.last_hard_state_id,
                                             host.problem_has_been_acknowledged,
                                             host.in_scheduled_downtime)
-        elif self.operand == 'service':
```
Comment (ddurieux) [inline alignak/dependencynode.py]: It's not pylint want, It only say, not use: 
```
if self.operand == 'host':
    return 'yes'
else:
    return 'no'
```
but use better:
```
if self.operand == 'host':
    return 'yes'
return 'no'
```
So you can use if and elif but remove the else when have return in if or elif

Comment (mohierf) [inline alignak/dependencynode.py]: Hmm... ok. IMHO it is lass readable but I will update this

File: alignak/dependencynode.py [hunk 2]
```diff
@@ -782,14 +781,14 @@ def get_host_filters(self, expr):
         flags, expr = match.groups()
         if "g" in flags:
             return [filter_host_by_group(expr)]
-        elif "r" in flags:
+        if "r" in flags:
```
Comment (ddurieux) [inline alignak/dependencynode.py]: You need re-add the `elif` in this file ;)

Comment (mohierf) [inline alignak/dependencynode.py]: Oops! I missed some...

**Pull request merged**